### DISCUSSION TITLE
Unifi installer broken

### DISCUSSION THREAD

**Discussion by jorisboth** [tteck/Proxmox#2576] [Help Needed] [answered]
Hi all,

I was trying to install the Unifi LXC but the script failed. Looking at the errors, I see that it is trying to download libssl on line 32:

wget -qL http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.21_amd64.deb

However, libssl1.1_1.1.1f-1ubuntu2.21_amd64.deb is replaced by libssl1.1_1.1.1f-1ubuntu2.22_amd64.deb yesterday on the 27th of February.

Can somebody please update the script?

Thanks!

**Comment by tteck** [comment]
fixed https://github.com/tteck/Proxmox/commit/7759ecf7ddc1bbd698140f3ab2434608253785a2

**Reply by jorisboth** [reply]
Thanks!

**Reply by tteck** [reply]
Thanks for reporting.

**Comment by tteck** [comment, marked as answer]
Even better, https://github.com/tteck/Proxmox/commit/7f02d0e51b8b3c4a631b4e1fce85914b2b5f810e
